lib/export-scala-regmap.js

```diff
@@ -486,30 +487,38 @@ const generateOMRegisterMap = (addressUnitBits, addressBlock) => {
   const registerMapElements = addressBlock.registers.map((register) => {
     const regFieldSeqElements = (register.fields || []).map(field => {
       const access = field.access || register.access || addressBlock.access;
-      return `${field.bitOffset} -> ` + generateRegisterField(field, 'Bool()', access, field.resetValue);
+      return field.bitOffset + ' -> ' +
+        generateRegisterField(field, 'Bool()', access, field.resetValue);
     });
 
-    const registerDesc = register.description ? `Some("""${register.description}""")` : 'None';
+    const registerDesc = register.description
+      ? `Some("""${register.description}""")`
+      : 'None';
 
     if (addressUnitBits % 8 != 0) {
       throw new Error('addressUnitBits must be a multiple of 8.');
     }
 
-    const addressOffset =
-      addressUnitBits === 8 ? register.addressOffset : `${register.addressOffset} * ${addressUnitBits / 8}`;
+    const addressOffset = (addressUnitBits === 8)
+      ? register.addressOffset
+      : `${register.addressOffset} * ${addressUnitBits / 8}`;
 
     return endent`
       ${addressOffset} -> RegFieldGroup("${register.name}", ${registerDesc}, padFields(
         ${regFieldSeqElements}))`;
   });
 
-  return endent`
-    OMRegister.convert(
-      ${registerMapElements.join(',\n')})`;
+  return (
+`RegFieldAddressBlock(
+  AddressBlockInfo(
+    "${addressBlock.name}",
+    baseAddress = ${addressBlock.baseAddress},
```

Comment:
```suggestion
    addressOffset = ${addressBlock.baseAddress},
```

This is what it currently is called in the rocket-chip PR. baseAddress isn't what it really is and rocket-chip uses addressOffset

Reply:
fixed